src/Data.sol

```diff
@@ -72,14 +70,14 @@ contract Data is IData, IndexResolver {
     function redeployIndex() public view onlyOwner {
         require (msg.value >= (_indexDeployValue * 2), DataErrors.value_less_than_required);
         tvm.accept();
-        tvm.rawReserve(msg.value, 1);
+
 
         address oldIndexOwner = resolveIndex(address(0), address(this), _addrOwner);
-        IIndex(oldIndexOwner).destruct();
+        IIndex(oldIndexOwner).destruct(); // Like tvm.commit
         address oldIndexOwnerRoot = resolveIndex(_addrRoot, address(this), _addrOwner);
```

Comment:
Why did you add this comment, can explain it?

Reply:
It's for you. What you don't need their tvm rawReserve